pkg/kubelet/status/status_manager.go

```diff
@@ -337,6 +337,17 @@ func (m *manager) updateStatusInternal(pod *v1.Pod, status v1.PodStatus, forceUp
 		initCondition.LastTransitionTime = lastTransitionTime
 	}
 
+	// Set PodScheduledCondition.LastTransitionTime.
```

Comment:
nit: There's a lot of copy-pasting happening here. Should we just factor this out as a function?

Reply:
Otherwise, the code lgtm

Reply:
Done